build_dependencies.sh

```diff
@@ -0,0 +1,61 @@
+#!bin/bash
+ExternalsFolder="./externals"
+DIDCommFolder="./externals/didcomm"
+AnonCredsFolder="./externals/anoncreds"
+GeneratedFolder="./generated"
+buildDIDComm() {
+    cd $DIDCommFolder/wasm
+    wasm-pack build --target=web --out-dir=../../../generated/didcomm-wasm-browser
+    wasm-pack build --target=nodejs --out-dir=../../../generated/didcomm-wasm-node
+    cd ../../../
+    rm ./didcomm.commit
+    git submodule | grep didcomm | awk '{print $1}' >> ./didcomm.commit
```

Comment:
could simplify this to remove the `rm` and redirect the output instead of appending:

```suggestion
    git submodule | grep didcomm | awk '{print $1}' > ./didcomm.commit
```